diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter2/comments/CommentsRule.kt

```diff
@@ -135,6 +136,9 @@ class CommentsRule(configRules: List<RulesConfig>) : DiktatRule(
         return codeFileStartCases.any { textWithoutCommentStartToken.contains(it) }
     }
 
+    private fun isContainsRequiredPartOfCode(text: String): Boolean =
```

Comment:
```suggestion
    private fun containsRequiredPartOfCode(text: String): Boolean =
```
Why not just `contains`?

Reply:
The rule is triggered to check for the presence of "is" in the name of the function that returns boolean type